tests/metagpt/test_incremental_dev.py

```diff
@@ -115,8 +116,11 @@ def get_incremental_dev_result(idea, project_name, use_review=True):
     if not project_path.exists():
         # If the project does not exist, extract the project file
         try:
-            # Use the tar command to extract the .zip file
-            subprocess.run(["tar", "-xf", f"{project_path}.zip", "-C", str(project_path.parent)], check=True)
+            if os.name == "nt":
+                subprocess.run(["tar", "-xf", f"{project_path}.zip", "-C", str(project_path.parent)], check=True)
+            else:
+                subprocess.run(["unzip", f"{project_path}.zip", "-d", str(project_path.parent)], check=True)
```

Comment:
I think a generic unzip function, or something like that, should be provided. This function should check if the command exists and then execute the command, a process that is not system-agnostic, but the command provided